src/inc/quic_platform_winkernel.h

```diff
@@ -264,8 +264,8 @@ CxPlatLogAssert(
 
 extern uint64_t CxPlatTotalMemory;
 
-#define CXPLAT_ALLOC_PAGED(Size, Tag) ExAllocatePool2(POOL_FLAG_PAGED | POOL_FLAG_UNINITIALIZED, Size, Tag)
-#define CXPLAT_ALLOC_NONPAGED(Size, Tag) ExAllocatePool2(POOL_FLAG_NON_PAGED | POOL_FLAG_UNINITIALIZED, Size, Tag)
+#define CXPLAT_ALLOC_PAGED(Size, Tag) ExAllocatePool2(POOL_FLAG_PAGED, Size, Tag)
```

Comment:
I know it's going to be more work, but we should do two things:

1. Update all the platform allocations to standardize the zero initialize.
2. Remove any unnecessary/additional zero'ing that is still there.

Reply:
Yeah I was planning to do that. Agreed yes its a lot of work.